### DISCUSSION TITLE
Laravel Migration with --safe flag

### DISCUSSION THREAD

**Discussion by manadinho** [laravel/framework#47523] [Ideas] [answered]
Few days back i created a PR. In that user can runs migrate command with --safe flag with this user ables to see the list of remaining migrations and control will ask user to do you want to run all these migrations or not. It's basically a safe check before run migration blindly. Is it helpful or not.

**Comment by TheBlckbird** [comment]
I think this would be really useful, especially when you rolled back migrations and are now unsure which migrations still need to be run

**Comment by TheBlckbird** [comment, marked as answer]
nvm, you can use `php artisan migrate:status` to see which migrations are already run an which are about to be run.

**Comment by manadinho** [comment]
Hmmm its helpful, thank you.
